### ISSUE TITLE
Zero exit code when failing setup

### ISSUE THREAD

**Issue by depsir** [loadimpact/k6#920]
I would like my test to fail, i.e. exit code greater than zero, if the setup fails.

With this minimal example you can see that the exit code is 0.
```
import { fail } from 'k6'
import http from 'k6/http'

export function setup(){
    fail()
}

export default function(){
    http.get('www.google.com')
}
```

```
$ k6 version                                                                                                                                                                             
k6 v0.23.0
```

**Comment by MStoykov** [comment]
Try the latest master it was fixed in https://github.com/loadimpact/k6/commit/a1df7bf91666c9fe367ba9932be37158f39de684 but is still not released. 

We are probably going to release a new version next week so then I am going to close this :)

**Comment by depsir** [comment]
Trying the minimal example with the latest version on master (https://github.com/loadimpact/k6/commit/9af59bc1ca2eee2f065a544c9400c08875c5d268) k6 exits with error 130

Hope gets released soon, along with new docker images

thanks

**Comment by na--** [comment]
We forgot to close this issue, the fix was released as part of k6 0.24.0: https://github.com/loadimpact/k6/blob/master/release%20notes/v0.24.0.md#bugs-fixed
